google/cloud/speech/client.py

```diff
@@ -0,0 +1,182 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Basic client for Google Cloud Speech API."""
+
+from base64 import b64encode
+
+from google.cloud._helpers import _to_bytes
+from google.cloud import client as client_module
+from google.cloud.speech.connection import Connection
+
+
+class Encoding(object):
+    """Audio encoding types.
+
+    See:
+    https://cloud.google.com/speech/reference/rest/v1beta1/\
+    RecognitionConfig#AudioEncoding
+    """
+
+    LINEAR16 = 'LINEAR16'
+    """LINEAR16 encoding type."""
+
+    FLAC = 'FLAC'
+    """FLAC encoding type."""
+
+    MULAW = 'MULAW'
+    """MULAW encoding type."""
+
+    AMR = 'AMR'
+    """AMR encoding type."""
+
+    AMR_WB = 'AMR_WB'
+    """AMR_WB encoding type."""
+
+
+class Client(client_module.Client):
+    """Client to bundle configuration needed for API requests.
+
+    :type project: str
+    :param project: The project which the client acts on behalf of. Will be
+                    passed when creating a dataset / job.  If not passed,
+                    falls back to the default inferred from the environment.
+
+    :type credentials: :class:`oauth2client.client.OAuth2Credentials` or
+                       :class:`NoneType`
+    :param credentials: The OAuth2 Credentials to use for the connection
+                        owned by this client. If not passed (and if no ``http``
+                        object is passed), falls back to the default inferred
+                        from the environment.
+
+    :type http: :class:`httplib2.Http` or class that defines ``request()``.
+    :param http: An optional HTTP object to make requests. If not passed, an
+                 ``http`` object is created that is bound to the
+                 ``credentials`` for the current object.
+    """
+
+    _connection_class = Connection
+
+    def sync_recognize(self, content, source_uri, encoding, sample_rate,
+                       language_code=None, max_alternatives=None,
+                       profanity_filter=None,
+                       speech_context=None):
+        """Synchronous Speech Recognition.
+
+        .. _sync_recognize: https://cloud.google.com/speech/reference/\
+                            rest/v1beta1/speech/syncrecognize
+
+        See `sync_recognize`_.
+
+        :type content: bytes
+        :param content: Byte stream of audio.
+
+        :type source_uri: str
+        :param source_uri: URI that points to a file that contains audio
+                      data bytes as specified in RecognitionConfig.
```

Comment:
This text should be lines up with `URI`. Or, if you want more space, move all the text below and indent 4 spaces.

```
        :param source_uri: 
            URI that points to a file that contains audio
            data ...
```